omnij-rpc/src/main/java/foundation/omni/rpc/OmniExtendedClient.java

```diff
@@ -103,10 +104,10 @@ public Sha256Hash acceptDexOffer(Address fromAddress, CurrencyID currencyId, Big
     public Sha256Hash createMetaDexSellOffer(Address address, CurrencyID currencyForSale, BigDecimal amountForSale,
                                       CurrencyID currencyDesired, BigDecimal amountDesired,
                                       Byte action) throws JsonRPCException, IOException {
-        Long willetsForSale = BTC.btcToSatoshis(amountForSale).longValue();  // Assume divisible property
-        Long willetsDesired = BTC.btcToSatoshis(amountDesired).longValue();  // Assume divisible property
+        Coin willetsForSale = BTC.btcToCoin(amountForSale);  // Assume divisible property
```

Comment:
Ah. This whole refactoring *started* with me trying to eliminate/reduce usage of the BTC class and eventually led to work on `OmniDivisibleValue`. Now I should be able to use `OmniDivisibleValue` here, too!